examples/src/main/java/com/cloudera/nav/sdk/examples/extraction/FilteredExtraction.java

```diff
@@ -61,8 +61,8 @@ public static void main(String[] args) throws IOException {
     MetadataExtractor extractor = new MetadataExtractor(client, null);
 
     // Run filtered examples
-    getHDFSEntities(client, extractor, marker);
-    getHive(extractor, marker, "city_id");
+    //getHDFSEntities(client, extractor, marker);
```

Comment:
Fixed. Some sources were returning null as the sourceExtractionIterationId and MetadataExtractor was not being able to handle that. Have added a check for that in MetadataExtractor